src/charselect.cpp

```diff
@@ -176,8 +176,10 @@ void Courtroom::char_clicked(int n_char)
         new AOPacket("CC#" + QString::number(ao_app->s_pv) + "#" +
                      QString::number(n_char) + "#" + get_hdid() + "#%"));
   }
-  else
+  else {
     update_character(n_char);
+    set_widgets(); // so we don't erroneously keep the charselect's fixedSize
```

Comment:
Wait, why wasn't this happening before? Don't think this is the right place to put this.

Reply:
i have no idea why this wasn't happening before, but set_widgets() is what sets the courtroom fixedSize and it's only called from enter_courtroom, which is only called the first time you select a character. calling it when we call update_character() fixes this

Reply:
Uhhh... okay.